README.md

```diff
@@ -1,9 +1,15 @@
-Follow steps below to use:
 
-Edit any details to customise the environment as you require in ansible-hosts-multimaster.
-
-Run:
+To execute the playbooks and set up your Openshift environment, run the following:
 
 ```
 ./deploy-openshift.sh
 ``` 
+
+The environment definition is in the file "/etc/ansible/group_vars/all.yml". This file is written by the ansible-heat code.
+(**Note** This file is a default location in Ansible, so there is no need to specify it when running playbooks; it will be loaded automatically.)
```

Comment:
We should test this assertion - my understanding was that since we're using a hosts file not in the default location we'd need to link to this from our working directory, hence when the code is checked out by openshift-heat it also creates a symlink to that folder from within /usr/share/ansible/openshift-deployment-ansible/group_vars - see [here](https://github.com/UKCloud/openshift-heat/blob/11bfcb59be75bbc8778c87c7b17a23f74445c7e9/files/setup_bastian.yaml#L101) if that helps. If that isn't necessary we should remove that from the other codebase.

Reply:
I will have to test that. 

My undertanding from [the Ansible documentation](http://docs.ansible.com/ansible/latest/intro_inventory.html#splitting-out-host-and-group-specific-data) is that the file will be picked up automatically.

Reply:
You were right - and I've updated the docs.